### PR TITLE
sha1 file issue fix

### PR DESCRIPTION
**Pull request by Vishal14583**
1. Mime type issue for ISO and SHA1 files resolved 
2. SHA1 file issue resolved

### CODE REVIEW

File: UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.API.FunctionalTests/Helpers/GetBatchDetails.cs [hunk 1]
```diff
@@ -91,8 +91,16 @@ public static void GetBatchDetailsResponseValidationForFullAVCSExchangeSet(dynam
                     string responseFileName = batchDetailsResponse.files[responseFileNameLocation].filename;
                     responseFileName.Should().Be(expectedFileName[responseFileNameLocation]);
 
-                    string responseFileMimeName = batchDetailsResponse.files[responseFileNameLocation].mimeType;
-                    responseFileMimeName.Should().Be(posDetails.IsoSha1FileMimeType);
+                    if (responseFileName.Contains(".sha1"))
```
[the remark this answers is not in the document]
Comment (Vishal14583) [reply UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.API.FunctionalTests/Helpers/GetBatchDetails.cs]: Code Replaced

File: UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Utilities/FileUtility.cs [hunk 2]
```diff
@@ -34,8 +34,10 @@ public void CreateISOImage(IEnumerable<string> srcFiles, string targetPath, stri
 
         public void CreateSha1File(string targetPath)
         {
-            byte[] isoFileBytes = System.Text.Encoding.UTF8.GetBytes(targetPath);
-            string hash = BitConverter.ToString(SHA1.Create().ComputeHash(isoFileBytes)).Replace("-", "");
+            FileInfo fileInfo = new(targetPath);
+            using Stream fs = fileInfo.OpenRead();
```
Comment (harshal11869) [inline UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Utilities/FileUtility.cs:38]:
```suggestion
            using Stream fileStream = fileInfo.OpenRead();
```

**Review by harshal11869 — CHANGES_REQUESTED** [review]

**Review by ajasonca — APPROVED** [review]
Looks good

File: UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Utilities/FileUtility.cs (hunk 2 above)
Comment (sumod13943) [reply UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Utilities/FileUtility.cs]: Changed

Comment (Vishal14583) [reply UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Utilities/FileUtility.cs]: Resolved

**Review by harshal11869 — APPROVED** [review]
Looks Good

**Pull request merged**